### ISSUE TITLE
Missing Footer

### ISSUE THREAD

**Issue by IPPYON596** [PreetySinha84/VerveBridge-Task-1-Weather-App#38]
Hello @PreetySinha84 👋,
I noticed that the website currently does not have a footer section. A footer is essential for usability and accessibility as it usually contains important information like:
Contact details
Copyright notice
Social media links
Navigation shortcuts
Privacy Policy / Terms links
I would like to work on this issue, This would improve the user experience and give the project a more polished, complete look.

**Comment by UttaraPraveen** [comment]
Hello, Im a GSSoC'25 contributor, i would like to work on this issue. Could you please assign it to me?
